stories/SQFormScrollableCardsMenuWrapper.stories.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import * as yup from 'yup';
+
+import {
+  SQFormScrollableCardsMenuWrapper,
+  SQFormScrollableCard,
+  SQFormTextField,
+  SQFormCheckbox
+} from '../src';
+import {createDocsPage} from './utils/createDocsPage';
+
+export default {
+  title: 'Forms/SQFormScrollableCardsMenuWrapper',
+  component: SQFormScrollableCardsMenuWrapper,
+  parameters: {
+    docs: {page: createDocsPage({showStories: false})}
+  }
+};
+
+export const sqFormScrollableCardsMenuWrapper = () => {
+  const menuItems = [
+    {
+      label: 'Details',
+      value: 'details'
+    },
+    {
+      label: 'Permissions',
+      value: 'permissions'
+    }
+  ];
+
+  return (
+    <div style={{width: '100%', height: '100%'}}>
+      <SQFormScrollableCardsMenuWrapper
+        title="[selected user]"
+        menuItems={menuItems}
+      >
+        <ScrollableDetails
+          id="details" // maps to menuItems[0].value
```

Comment:
I think we can avoid passing in menuItems and an `id` prop that matches an array item from menuItems.
We can absorb the menuItems pattern into the `children`

```javascript
      <SQFormScrollableCardsMenuWrapper
        title="[selected user]"
      >
      <ScrollableDetails label="Details" value="details" />
      {/* etc, etc for each child */}
```

Then inside our SQScrollableCardsWrapper we can build the menuItems array dynamically.

```javascript
const menuItems = React.Children.map(child => {
  return {label: child.props.label, value: child.props.value}
})
```

Reply:
> I think we can avoid passing in menuItems and an `id` prop that matches an array item from menuItems.
> We can absorb the menuItems pattern into the `children`
> 
> ```js
> // pseudo code
> ```
> 
> ```js
>       <SQFormScrollableCardsMenuWrapper
>         title="[selected user]"
>       >
>       <ScrollableDetails label="Details" value="details" />
>       {/* etc, etc for each child */}
>       </SQFormScrollableCardsMenuWrapper>
> ```
> 
> Then inside our SQScrollableCardsWrapper we can build the menuItems array dynamically.
> 
> ```js
> const menuItems = React.Children.map(child => {
>   return {label: child.props.label, value: child.props.value}
> })
> ```
> 
> * Edit to add obligatory pseudo code warning

I like this idea! Should work well.

Reply:
done